benchmarks/video-analytics-standalone/python/requirements.txt

```diff
@@ -0,0 +1,16 @@
+opencv-python-headless
+torch
```

Comment:
I think it would be good practice to fix all the dependencies.
`pip freeze > requirements.txt` should do the magic.